modules/vaos/app/models/vaos/appointment_request_form.rb

```diff
@@ -133,25 +133,25 @@ def patient_identifier
     end
 
     def first_name
-      @user.mpi&.profile&.given_names&.first
+      @user.given_names&.first
```

Comment:
Can we not just delegate these to user? I'm not sure how that works across modules / components.

Reply:
I only kept `given_names.first` to attempt to keep behavior consistent (if we request `first_name` from the user, it will first try to grab the name off of the `UserIdentity` object, and if that doesn't exist, grab it off the MPI object in a similar way to how it does it here). However, since I've decided to split this PR, I think I have more stomach for the small changes in behavior with letting `User` figure out how to retrieve the field. I'll change this to `user.first_name` and perhaps in a specific PR figure out if this changes anything